src/transformers/models/auto/modeling_auto.py

```diff
@@ -484,6 +484,7 @@
         ("deberta-v2", "DebertaV2ForTokenClassification"),
         ("gpt2", "GPT2ForTokenClassification"),
         ("ibert", "IBertForTokenClassification"),
+        ("luke", "LukeForEntitySpanClassification"),
```

Comment:
Pinging @Narsil here. Won't this cause a problem in the pipeline tests? Since LUKE handles NER quite differently compared to `xxxForTokenClassification` models.

Reply:
It might, the fast pipeline test should tell us that pretty rapidly.

We *can* add exceptions for some models when they can't work on pipelines for good reasons and we still want them in the correct ForXX bucket.

The `get_test_pipeline` function in those tests should be used to trigger the `skip`.
But we need a good comment saying why the test is being skipped, so future us can revisit the decision (should we make the pipeline support this odd behavior or not, that sort of thing)

Reply:
In that case, I would remove LUKE from the Auto API.

Reply:
Seconded, it's not even necessary for the example to work since the example should use the specific model class.